test/test-mint.ts

```diff
@@ -1,28 +1,27 @@
-import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
-import { expect } from "chai";
-import { Contract } from "ethers";
-import { ethers } from "hardhat";
-import { Deployment } from "hardhat-deploy/dist/types";
+/* eslint-disable no-unused-expressions */
```

Comment:
why? we don't want unused stuff no?

Reply:
without it it's complaining at `.to.be.true`
🤷🏼‍♂️
fucking lint with this is so bad

Reply:
yea I agree, maybe add it on the lines tho.